### ISSUE TITLE
Unable to do any Medic if you don't have items in your gear. 

### ISSUE THREAD

**Issue by ThecMaster** [KoffeinFlummi/AGM#2056]
I have noticed that you can't do any Medic tasks except diagnosing if you don't have that item in gear. The player you are for example are bandaging can have more than enough bandages but you can't do anything if you don't have any. But you can do them if you got at least one. 

Our settings are that you don't need to be a medic, and items are taken form the one that receiving the treatment. Therefor you should be able to bandage someone else without having bandages in your own gear. There should be enough with at least one item in your gear OR in the receiving players gear. 

Sorry for my English. Hope you understand anyway. And sorry if this is a double post. I have searched for an hour before I posted this. I couldn't find anything about it. 

MVH// ThecMaster

**Comment by bux578** [comment]
Thank you for your report.

That's totally true.

**Comment by ThecMaster** [comment]
Thanks for fast response. We just need to wait for 0.96 then. :D

**Comment by bux578** [comment]
On the other hand, it's always advisable to hand out medical equipment to everyone (like IRL).

**Comment by ThecMaster** [comment]
Yeah we all have bandages. But in the field when you used all of your bandages you can't help anyone without putting bandages in your gear first. That takes time and that time can be the difference between life and death. :D
